src/components/HeaderWithLanguage/HeaderStatusInformation.tsx

```diff
@@ -14,7 +14,7 @@ import Tooltip from '@ndla/tooltip';
 import config from '../../config';
 import LearningpathConnection from './LearningpathConnection';
 import EmbedConnection from './EmbedInformation/EmbedConnection';
-import { Learningpath } from '../../interfaces';
+import { Learningpath } from '../../modules/learningpath/learningpathApiInterfaces';
```

Comment:
Burde ikkje dei nye typene vore brukt også i module interfaca?

Reply:
Vet ikke helt om jeg henger med. Har bare skrevet om typene for SubjectPage og Film, har ikke rørt noen andre typer ennå. Tenkte å ta det i en annen PR.